jetbrains_plugin/src/main/kotlin/com/sina/weibo/agent/extensions/common/ExtensionType.kt

```diff
@@ -8,6 +8,7 @@ enum class ExtensionType(val code: String, val displayName: String, val descript
     ROO_CODE("roo-code", "Roo Code", "AI-powered code assistant"),
     CLINE("cline", "Cline AI", "AI-powered coding assistant with advanced features"),
     KILO_CODE("kilo-code", "Kilo Code", "AI-powered code assistant with advanced capabilities"),
+    COSTRICT("costrict", "Costrict", "AI-powered code assistant with advanced capabilities"),
```

Comment:
在ExtensionType.kt文件中，枚举值的命名可能有拼写错误。应该使用COSTRICT还是COCONSTRICT？

```suggestion
ROO_CODE("roo-code", "Roo Code", "AI-powered code assistant"),
    CLINE("cline", "Cline AI", "AI-powered coding assistant with advanced features"),
    KILO_CODE("kilo-code", "Kilo Code", "AI-powered code assistant with advanced capabilities"),
    COCONSTRICT("costrict", "Costrict", "AI-powered code assistant with advanced capabilities"),